q2_katharoseq/plugin_setup.py

```diff
@@ -60,3 +69,44 @@
                 'positive signal in samples with as few as 50 to 500 cells.',
     citations=[citations['minich2018']]
 )
+
+
+plugin.methods.register_function(
+    function=estimating_biomass,
+    inputs={},
+    parameters={'total_reads': MetadataColumn[Numeric],
+                'control_cell_extraction': MetadataColumn[Numeric],
+                'positive_control_column': MetadataColumn[Categorical],
+                'positive_control_value': Str,
+                'extraction_mass_g': MetadataColumn[Categorical],
+                'min_total_reads': Int,
+                'pcr_template_vol': Int,
+                'dna_extract_vol': Int},
+    outputs=[('estimated_biomass', EstimatedBiomass)],
+    input_descriptions={},
+    parameter_descriptions={
+        'total_reads': 'The total reads present in each sample.',
+        'control_cell_extraction': 'The number of cells in the controls.',
```

Comment:
'The estimated number of cells or genomes used as input to your library prep. One may typically estimate this by determining the total number of cells from a stock solution used to make standard titrations. Each titration will have an estimated number of microbial cells put into the extraction. The final estimate will depend on the elution volume and the final volume used into the library prep (e.g. 16S PCR).'